go.mod

```diff
@@ -204,7 +204,7 @@ require (
 	k8s.io/apiserver v0.23.8
 	k8s.io/autoscaler/vertical-pod-autoscaler v0.10.0
 	k8s.io/client-go v0.23.8
-	k8s.io/cri-api v0.23.8
+	k8s.io/cri-api v0.25.0
```

Comment:
Ideally, if you bump one, you should bump all aligned `k8s.io` version (even though I believe `cri-api` is mostly standalone)

Reply:
Ah you're right, I hoped that my change would not be that big